filament/backend/src/metal/MetalDriver.mm

```diff
@@ -55,7 +55,8 @@
 MetalDriver::MetalDriver(backend::MetalPlatform* platform) noexcept
         : DriverBase(new ConcreteDispatcher<MetalDriver>()),
         mPlatform(*platform),
-        mContext(new MetalContext) {
+        mContext(new MetalContext),
+        mHandleAllocator("Handles", FILAMENT_OPENGL_HANDLE_ARENA_SIZE_IN_MB * 1024U * 1024U) {
```

Comment:
I created a new constant for VULKAN so we might as well do the same for Metal, at least for now, and make it the same value.  Otherwise it's strange to refer to an OPENGL thing here.

Reply:
Yeah, sounds good.